.npmignore

```diff
@@ -0,0 +1,13 @@
+.rollupcache
```

Comment:
Any specific reason why we do not want to deliver the src files? So far I always liked the idea of doing that because consumers can easily navigate to the node_modules/aurelia-store repo, fix stuff and rebuild for trying out vs having to clone the repo, fix stuff and link locally.

Reply:
should `.circleci` and `doc` be excluded as well?

Reply:
I think it's all or nothing, so if we are to support that, probably just don't ignore anything beside CI

Reply:
the use-case I mentioned is nice, but definitely not mandatory. So I guess adding circleci and doc should be good